doc/authors.rst

```diff
@@ -6,6 +6,7 @@
 .. _Anand Saini: https://github.com/anandsaini024
 .. _Ariel Rokem: https://github.com/arokem
 .. _Austin Hurst: https://github.com/a-hurst
+.. _Berk Gercek: https://github.com/berkgercek
```

Comment:
doc build failure is this:
```
/home/runner/work/mne-bids/mne-bids/doc/whats_new.rst:21: ERROR: Unknown target name: "berk gerçek". [docutils]
/home/runner/work/mne-bids/mne-bids/doc/whats_new.rst:37: ERROR: Unknown target name: "berk gerçek". [docutils]
```
which I think will be fixed by 

```suggestion
.. _Berk Gerçek: https://github.com/berkgercek
```

Reply:
Whoops, and thank you! Didn't realize I added myself retroactively to an existing paper. My mistake.